### PR TITLE
feat(backend): Configuring GridFs in spring boot application for hand…

### PR DESCRIPTION
**Pull request by mbunwe-victor**

### CODE REVIEW

**Review by bryandino673 — APPROVED** [review]

**Review by Christiantyemele — COMMENTED** [review]
@stephane-segning and @mbunwe-victor i think this should have a DBObject which will implement store methods

**Comment by mbunwe-victor** [comment]
> @stephane-segning and @mbunwe-victor i think this should have a DBObject which will implement store methods

@Christiantyemele, the implementation of GridFsTemplate handles everything that has to do with the store method.The GridFsTemplate encapsulates the functionality needed to interact with GridFS for file storage

**Review by ValantineSuh — APPROVED** [review]

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Please write some unit tests and integration tests to ensure this is working

**Comment by mbunwe-victor** [comment]
> Please write some unit tests and integration tests to ensure this is working

Done

**Review by ValantineSuh — APPROVED** [review]
Looks good!

File: src/test/java/com/ssegning/lynx/lynxbackend/GridFsConfigurationTest.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    @DisplayName("")
+    public void testMongoClient() {
+
+
+        // Call methods and perform assertions
+        MongoClient returnedMongoClient = gridFsConfiguration.mongoClient();
+
+        assertNotNull(returnedMongoClient);
+        assertEquals(mockMongoClient, returnedMongoClient); // Verify the returned MongoClient is the same as the mocked one
+    }
```
Comment (stephane-segning) [inline src/test/java/com/ssegning/lynx/lynxbackend/GridFsConfigurationTest.java:48]: This is not testing something. Remove this test

File: src/test/java/com/ssegning/lynx/lynxbackend/GridFsConfigurationTest.java
Comment (stephane-segning) [inline src/test/java/com/ssegning/lynx/lynxbackend/GridFsConfigurationTest.java:1]: This test case is not useful. Please remove it

**Review by stephane-segning — CHANGES_REQUESTED** [review]

File: src/test/java/com/ssegning/lynx/lynxbackend/GridFsConfigurationTest.java
Comment (mbunwe-victor) [reply src/test/java/com/ssegning/lynx/lynxbackend/GridFsConfigurationTest.java:1]: I have remove the GridFsConfigurationTest.java

**Review by stephane-segning — APPROVED** [review]

**Review by Christiantyemele — APPROVED** [review]

**Pull request merged**